### PR TITLE
adjust lockup gas limits

### PR DESCRIPTION
**Pull request by sparrowDom**
adjust lockup gas limits

### CODE REVIEW

File: client/components/vote-escrow/LockupForm.tsx [hunk 1]
```diff
@@ -302,8 +302,8 @@ const LockupForm: FunctionComponent<LockupFormProps> = ({ existingLockup }) => {
         transaction = await contracts.OgvStaking["extend(uint256,uint256)"](
           existingLockup.lockupId,
           duration,
-          { gasLimit: 1000000 }
-        ); // @TODO maybe set this to lower
+          { gasLimit: 154727 }
```
Comment (franckc) [inline client/components/vote-escrow/LockupForm.tsx:305]: This is tight.
Tom did an extend on mainnet
https://etherscan.io/tx/0x3f0ba0620459079f8b2d55c380f2409aa876e8776c2b481281af09061ebebb3a

It used 149,604 

How about 150k*1.3 = 195k gas?

File: client/components/vote-escrow/LockupForm.tsx [hunk 2]
```diff
@@ -188,8 +188,8 @@ const LockupForm: FunctionComponent<LockupFormProps> = ({ existingLockup }) => {
       transaction = await contracts.OriginDollarGovernance.approve(
         contracts.OgvStaking.address,
         ethers.constants.MaxUint256,
-        { gasLimit: 1000000 }
-      ); // @TODO maybe set this to lower
+        { gasLimit: 66983 }
```
Comment (franckc) [inline client/components/vote-escrow/LockupForm.tsx:191]: That looks too tight. We did an approve from the multisig and it used 149,604
https://etherscan.io/tx/0x3f0ba0620459079f8b2d55c380f2409aa876e8776c2b481281af09061ebebb3a

**Comment by tomhirst** [comment]
We are missing a figure for `unstake` transactions. I guess users won't be able to call this until at least 1 month has passed. We should be OK to leave for now, but we should remember that we need to test and add a more accurate amount.

File: client/components/vote-escrow/LockupForm.tsx (hunk 2 above)
Comment (franckc) [reply client/components/vote-escrow/LockupForm.tsx:191]: Wrong hash. Here is the one:
https://etherscan.io/tx/0x059faaef43ca156e2da0624bf63a04cd3cb841f1da11c6772f70dd0577b38b17
 
Using 111,065

**Review by franckc — APPROVED** [review]
LGTM

**Pull request merged**